pet-store-react-hooks/src/DeletePetModal/DeleteModal.test.tsx

```diff
@@ -0,0 +1,203 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { HttpResponse, http } from 'msw';
+import { setupServer } from 'msw/node';
+
+import { DeletePetModal } from '~/DeletePetModal/DeletePetModal';
+import { apiBaseUrl } from '~/infrastructure/api-client';
+import { WaitHandle } from '~testing/WaitHandle';
+import { petKindsMap, pets, petsList } from '~testing/data';
+import { handlers } from '~testing/handlers';
+
+const server = setupServer(...handlers);
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+// Question: When i have pet for testing, is it OK to get it outside of the tests and use it in all of them?
```

Comment:
yes, or you can just define a fresh one here that doesn't come from ~testing/data and use that.